var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -67,6 +68,9 @@ class PyPsyclone(PythonPackage):
     depends_on("py-pytest-xdist", type="test")
     depends_on("py-pytest", type="test")
 
+    # Maintainers
+    maintainers("arporter", "sergisiso")
+
```

Comment:
Please move this directive towards the top, such as before or after the license.

Reply:
Done.